cloud_governance/policy/aws/unused_nat_gateway.py

```diff
@@ -25,6 +25,8 @@ def __check_cloud_watch_logs(self, resource_id: str, days: int = UNUSED_DAYS):
         :param days:
         :return:
         """
+        if days == 0:
+            days = 1
```

Comment:
Is it ok to revert it back to 1 ?

Reply:
we need to always check for the last one day.

Reply:
we need to always check for the last one day.